packages/core/src/components/List/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import childrenWithComponentName from '../../prop-types/childrenWithComponentName';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import Item from './Item';
+
+export { Item };
+
+export type Props = {
+  /** List items. */
+  children: NonNullable<React.ReactNode>;
+  /** Flex align the items. */
+  flexAlign?: boolean;
+  /** Renders an `<ol></ol>`. */
+  ordered?: boolean;
+};
+
+class List extends React.Component<Props & WithStylesProps> {
+  static propTypes = {
+    children: childrenWithComponentName('ListItem?|li?').isRequired,
```

Comment:
Dont think the `?` in the regex is required.

Reply:
will test it out! pulled this from some other component